sources/shdr/App.coffee

```diff
@@ -186,6 +186,18 @@ class App
             shdr.UI.WARNING)
         console.warn 'ERROR: ', e
 
+  upload: (file_obj) ->
+    try
+      reader = new FileReader()
+      reader.readAsDataURL file_obj
```

Comment:
nit: parens on function call and camelCase variables.
`reader.readAsDataURL(fileObj)`